libs/server/TLS/GarnetTlsOptions.cs

```diff
@@ -161,21 +161,69 @@ SslClientAuthenticationOptions GetSslClientAuthenticationOptions()
             };
         }
 
+
+        /// <summary>
+        /// Callback to verify the TLS certificate
+        /// </summary>
+        /// <param name="issuerCertificatePath"></param>
```

Comment:
NIT: Incomplete documentation comments. (I know the current code is not very consistently documented either, but we should try to have complete commends for all new functions)